### PR TITLE
Added Penny to the list of Solid Apps

### PR DESCRIPTION
**Pull request by nicolasmondada**
Added Penny from Vincent Turu to the list of Solid Apps

### CODE REVIEW

File: pages/use-solid/apps.md [hunk 1]
```diff
@@ -144,10 +144,11 @@ redirect_from:
...
 | [Assemblage](https://gitlab.com/omd_public/assemblage) | A composable data browser aimed to provide a great user experience for Solid POD services, as well as other RDF sources, both for desktop and mobile web browsers. [Source code](https://gitlab.com/omd_public/assemblage) and [demo video](https://youtu.be/hYFEBAdkzmc). |
 | [Warp](https://linkeddata.github.io/warp/)                                   | File browser. <br>- [Source code](https://github.com/linkeddata/warp) [MIT License Copyright (c) 2014](https://github.com/linkeddata/warp/blob/gh-pages/LICENSE) [Andrei Samba](https://github.com/deiu) |
+| [Penny](https://penny.vincenttunru.com/)                                   | File browser. <br>- [Source code](https://github.com/linkeddata/warp) [MIT License Copyright (c) 2014](https://github.com/linkeddata/warp/blob/gh-pages/LICENSE) [Vincent Turu](https://gitlab.com/Vinnl) |
```
Comment (kay-kim) [inline pages/use-solid/apps.md:147]: Hey Nick -- 
this row and the below row seems to have gotten somehow entangled.

This Penny row seems to link to the Warp application  whereas below, in the Historical Solid Apps section, you seem to have clobbered Mark Book source and license link with Vincent's

**Review by kay-kim — CHANGES_REQUESTED** [review]

**Comment by KyraAssaad** [comment]
@Vinnl Penny is listed on https://solidproject.org/developers/tools/, do you want it listed on both pages? 

@MarrelleBailey @justinwb I'm making a note of this for the redesign - what's the difference between `Tools and Libraries` vs `Apps`?

**Comment by Vinnl** [comment]
@KyraAssaad No, I'm fine with it not being on the apps page; I'm explicitly "advertising" it as a developer tool rather than a user-facing app. I don't think many non-developer users would be happy using Penny or otherwise browsing raw data on their Pod.

**Comment by KyraAssaad** [comment]
Closing this since Penny is listed on https://solidproject.org/developers/tools/ already!

**Comment by jeff-zucker** [comment]
I'm the one who came up with the original breakout - tools, libraries, and apps.  At one point I suggested lumping tools back in with apps and would still be fine with that.  My mental breakdown is this : a library is something that is can not be used directly but is called by an app.  A tool is something developers use to manage their apps and data.  Molid fits that and is of little use to anyone but developers.  Penny and Solid-IDE and SolidShell I think can be used by users and so probably belong with Apps, especialy if there is a "Pod Management" section within apps.  Data-Kitchen includes the kitchen sink so is both a tool and a super-app.   I don't have a strong opinion but I guess I'd move tools to apps.

**Comment by KyraAssaad** [comment]
Thanks for this info, @jeff-zucker! That makes sense. I think the confusing part for me is that "Tools and Libraries" are together but "Apps" is a separate page. We'll probably keep the designation you explained, but maybe reorganize the pages so it's more clear

**Pull request closed without merging**